### ISSUE TITLE
Add ability to return the number of rows in a sheet

### ISSUE THREAD

**Issue by cfsimplicity** [cfsimplicity/lucee-spreadsheet#129]
Should blank rows be included?

**Comment by murpg** [comment]
I would use this feature to not have to hard code the rows argument to the spreadsheet.read function. I am reading spreadsheets and displaying it as html. When new rows are added i need to know how many are added. These rows are being added manually to the excel file.

**Comment by cfsimplicity** [comment]
Why do you need to use the `rows` argument of `read()` if you are reading all rows? Or are you starting from a particular row and reading the remainder? That would make sense.

**Comment by murpg** [comment]
I am starting from a particular row and reading the rest. I would also like to store the original row and look to determine if that has changed. If it has changed I would take action on that. For example if there was a typo there would be no change in rows. So, I would not want to take action on that. However if rows are added I am interested in that.

**Comment by cfsimplicity** [comment]
George, I've had a go at implementing `getRowCount()` in the *develop* branch. Could you test it please and let me know if it's ok?

The syntax is:
```
workbook = spreadsheet.read( filePath );
numberOfRows = spreadsheet.getRowCount( workbook );
```
You can optionally specify the `sheetNameOrNumber` as a second argument if there are multiple sheets.

**Comment by murpg** [comment]
Hi Thanks for doing this. I will report back.

**Comment by murpg** [comment]
Hi Julian, this works nicely and is ready to be merged into master. I have tested it for row inserts, deletes, and updates and it works as expected. Thanks for the great turn around on this. George Murphy

**Comment by cfsimplicity** [comment]
Thanks for testing, George.  All merged into master as v1.6.0.

**Comment by murpg** [comment]
Thx! Julian
